bindata/etcd/cluster-restore.sh

```diff
@@ -96,10 +96,7 @@ BACKUP_FILE=$(ls -vd "${BACKUP_DIR}"/static_kuberesources*.tar.gz | tail -1) ||
 SNAPSHOT_FILE=$(ls -vd "${BACKUP_DIR}"/snapshot*.db | tail -1) || true
 
 ETCD_STATIC_POD_LIST=("etcd-pod.yaml")
-AUX_STATIC_POD_LIST=("kube-apiserver-pod.yaml" "kube-controller-manager-pod.yaml" "kube-scheduler-pod.yaml")
-
-ETCD_STATIC_POD_CONTAINERS=("etcd" "etcdctl" "etcd-metrics" "etcd-readyz")
-AUX_STATIC_POD_CONTAINERS=("kube-controller-manager" "kube-apiserver" "kube-scheduler")
```

Comment:
Why are we not starting the auxillary static pods like the apiserver anymore? Doesn't our existing restore process depend on that as we restore to a single member control-plane node?
Or is that being handled elsewhere now?

Reply:
why would we need this when we bump the revisions? The caches are wiped by the watches expiring through the compaction marker.

Reply:
Ah my bad, I see the revision bump is on by default now with `ETCD_ETCDCTL_RESTORE_ENABLE_BUMP` gone.

So with this change we'll also need to update our [default DR workflow](https://docs.openshift.com/container-platform/4.16/backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.html#dr-scenario-2-restoring-cluster-state_dr-restoring-cluster-state) docs as well  I think to exclude that part.

```
The cluster-restore.sh script must show that etcd, kube-apiserver, kube-controller-manager, and kube-scheduler pods are stopped and then started at the end of the restore process.
```

Reply:
yep, I'm currently rewriting the entire doc for it

Reply:
Apologies if I'm conflating this with the newer quorum-restore/force-new-cluster approach, but for our standard existing workflow do we still need to stop the auxillary control-plane pods on the other non-recovery hosts if we have the revision bump enabled by default?

i.e the following pre-recovery step:
https://docs.openshift.com/container-platform/4.16/backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.html#dr-scenario-2-restoring-cluster-state_dr-restoring-cluster-state
```
4. Stop the static pods on any other control plane nodes.
...
d. Move the existing kube-apiserver file out of the kubelet manifest directory by running:

$ sudo mv -v /etc/kubernetes/manifests/kube-apiserver-pod.yaml /tmp
....
```
I'm guessing the API server and other pods would have their caches wiped so probably no need to restart them.

Normally we stop those but I forget if these non-recovery etcd pods also crash-loop after we restore the recovery member?

Trying to tally all the doc changes we'll need after this change.

Reply:
checkout
https://docs.google.com/document/d/1oIlHD-gkkskQUNN7n1EWVq5vi3UCOWJHuY3QEzDKsYY/edit#heading=h.7ebq38ydvskj

Reply:
>  for our standard existing workflow do we still need to stop the auxillary control-plane pods on the other non-recovery hosts if we have the revision bump enabled by default?

we will only have to stop etcd, everything else should fall into place. That's only for the restore from snapshot, quorum restore will automatically do that for us by stopping raft.

> Normally we stop those but I forget if these non-recovery etcd pods also crash-loop after we restore the recovery member?

when restoring from snapshot: no, 
when restoring with force-new-cluster: yes.